OpenRA.Mods.Common/Activities/Repair.cs

```diff
@@ -97,7 +99,7 @@ public override Activity Tick(Actor self)
 			if (remainingTicks == 0)
 			{
 				var unitCost = self.Info.TraitInfo<ValuedInfo>().Cost;
-				var hpToRepair = repairsUnits.Info.HpPerStep;
+				var hpToRepair = repairable != null ? repairable.Info.HpPerStep : repairsUnits.Info.HpPerStep;
```

Comment:
You can then mention this fallback as a Desc() in your own HpPerStep.

Reply:
I was thinking of using a nullable int for the HpPerStep on the Repairable trait, Would it be better to use < 0 instead? I couldnt find any traits using nullable  value types for properties in the code.

I also just saw that the helipad repair functionality doesn't use the Repair trait at all so I will need to fix handle that. 

Thanks!

Reply:
Notsure if you can unset a value properly from the yaml if you're overriding something from inheritance + nullable with a default value is already considered set. Nullable just doesn't worth itself in this environment IMO.

TBH, this needs more thought in general, because I'm notsure if cost should also scale along with the per-unit repair value.